### DISCUSSION TITLE
大佬，在VPS上安装ddns-go如何开启公网访问

### DISCUSSION THREAD

**Discussion by BattlefieldFlying** [jeessy2/ddns-go#960] [Q&A]
大佬，在VPS上安装ddns-go如何开启公网访问，Linux无图形界面，安装后无法通过公网访问[]:9876打开网页配置，有没有办法通过命令行修改ddns-go的配置文件开启公网访问

**Comment by jeessy2** [comment]
修改位于当前用户目录的.ddns-go-config.yaml文件

**Reply by BattlefieldFlying** [reply]
谢谢大佬
